cetmix_tower_aws/readme/CONFIGURE.md

```
@@ -0,0 +1,42 @@
+**Prerequisites**
+
+The module has `boto3` defined in its external dependencies, which means, you should install the Python `boto3` package manually if you don't have automatic package installation configured in your Odoo environment. Run `pip install boto3` to install it.  
+
+**Setting up AWS Access**
+
+1. **Create AWS Access Keys**
+
+   To use the AWS integration with Cetmix Tower, you need to create AWS access keys:
+   
+   - Follow the official AWS documentation https://docs.aws.amazon.com/IAM/latest/UserGuide/security-creds.html for creating IAM access keys
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)